tools/filters/catWrapper.xml

```diff
@@ -1,15 +1,15 @@
-<tool id="cat1" name="Concatenate datasets" version="1.0.0">
+<tool id="cat1" name="Concatenate datasets" version="1.0.1">
     <description>tail-to-head</description>
     <command interpreter="python">
         catWrapper.py
-        $out_file1
-        $input1
+        "$out_file1"
+        "${'" "'.join(str($input1).split(','))}"
```

Comment:
```
#for $i in $input1# "$i" #end for#
```

Should be considered a preferable construct, it doesn't cast the list into a string and then rip it backout into a list again - also if the path contains a comma it will be less incorrect.